index.php

```diff
@@ -49,36 +49,78 @@
         </nav>
     </div>
 </header>
-
+<?php $categories = ['Доски и лыжи', 'Крепления', 'Ботинки', 'Одежда', 'Инструменты', 'Разное']; 
+$products = [
+    [
+     'name' => '2014 Rossignol District Snowboard',   
+     'category' => 'Доски и лыжи',
+     'price' => '10999',
+     'pic_url' => 'img/lot-1.jpg'],
+    [
+     'name' => 'DC Ply Mens 2016/2017 Snowboard',   
+     'category' => 'Доски и лыжи',
+     'price' => '159999',
+     'pic_url' => 'img/lot-2.jpg'],
+    [
+     'name' => 'Крепления Union Contact Pro 2015 года размер L/XL',   
+     'category' => 'Крепления',
+     'price' => '8000',
+     'pic_url' => 'img/lot-3.jpg'],
+    [
+    'name' => 'Ботинки для сноуборда DC Mutiny Charocal',   
+    'category' => 'Ботинки',
+    'price' => '10999',
+    'pic_url' => 'img/lot-4.jpg'],
+    [
+    'name' => 'Куртка для сноуборда DC Mutiny Charocal',   
+    'category' => 'Одежда',
+    'price' => '7500',
+    'pic_url' => 'img/lot-5.jpg'],
+    [
+    'name' => 'Маска Oakley Canopy',   
+    'category' => 'Разное',
+    'price' => '5400',
+    'pic_url' => 'img/lot-6.jpg'],       
+    
+]
+?>
 <main class="container">
     <section class="promo">
         <h2 class="promo__title">Нужен стафф для катки?</h2>
         <p class="promo__text">На нашем интернет-аукционе ты найдёшь самое эксклюзивное сноубордическое и горнолыжное снаряжение.</p>
         <ul class="promo__list">
             <!--заполните этот список из массива категорий-->
-            <li class="promo__item promo__item--boards">
-                <a class="promo__link" href="pages/all-lots.html">Имя категории</a>
-            </li>
+        <?php $index = 0;
+         $key_num = count($categories);
+         while ($index < $key_num): ?>
+         <li class="promo__item promo__item--boards">
+            <a class="promo__link" href="pages/all-lots.html"><?=$categories[$index];?></a>
+         </li>
+        <?php $index++; ?>
+        <?php endwhile; ?>
         </ul>
+        
     </section>
     <section class="lots">
         <div class="lots__header">
             <h2>Открытые лоты</h2>
         </div>
         <ul class="lots__list">
             <!--заполните этот список из массива с товарами-->
+            <?php foreach ($products as $key => $val): ?>
```

Comment:
```suggestion
            <?php foreach ($products as $product): ?>
```
упраздни связку key => val. ты используешь только val. Можно использовать вот такую связку. Обращаться к значению теперь нужно так 
`$product['pic_url']`